index.html

```diff
@@ -104,126 +105,12 @@ <h2>I'am a Software Developer</h2>
         </div>
         <div class="background"></div>
       </section>
+      <!-- Work modal -->
+      <section id="work-modal">
```

Comment:
[OPTIONAL]
- [ ] Great work implementing the requirements for this project, however, I can see that in the mobile version the image of one of your modal popups is out center, please let's fix this.
![image](https://user-images.githubusercontent.com/28109626/156272123-a91ced5a-c586-4762-b699-daf87a11b5fa.png)

Reply:
Will do. 😎👍